.github/workflows/continuous-integration.yml

```diff
@@ -24,15 +31,38 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Cache dependencies
+      - name: Cache dependencies (Linux)
+        if: startsWith(runner.os, 'Linux')
         uses: actions/cache@v2
         with:
           path: ~/.cache/pip
           # Cache based on OS, Python version, and dependency hash
-          key: pip-${{ runner.os }}-python${{ matrix.python-version }}-${{ hashFiles('requirements-dev.txt') }}
+          key: test-${{ runner.os }}-python${{ matrix.python-version }}-${{ hashFiles('requirements-test.txt') }}
```

Comment:
I personally would prefer the original "pip" prefix, since you might need more than one type of cache later ([example](https://github.com/linz/geostore/blob/83a29363f1a6bf74f2dddfbd19e4d68b2c4d36b0/.github/workflows/test.yml#L69-L97)), for example in a multi-stage transition to another Python package manager, and then it's good to be specific.

Reply:
That's a good point, I'll update this.

Reply:
Fixed in a858c4cf2deeffa8eba2d5540d6b7864e26da996